src/thread0_epc.c

```diff
@@ -90,19 +89,20 @@ void func_800B6F04(void) {
     D_80127CAE += 11;
 }
 
-void func_800B6F30(s32 arg0, s32 arg1, s32 arg2) {
+void func_800B6F30(UNUSED s32 arg0, UNUSED s32 arg1, UNUSED s32 arg2) {
 }
 
-void func_800B6F40(s32 arg0, s32 arg1, s32 arg2) {
+void func_800B6F40(UNUSED s32 arg0, UNUSED s32 arg1, UNUSED s32 arg2) {
 }
 
 #ifdef NON_EQUIVALENT
-void func_800B6F50(void) {
+void thread0_create(void) {
     s32 i;
 
     // Almost matching, just have an issue with argument 4.
     // The -O2 compiler is too smart. :(
-    osCreateThread(&D_801295E0, 0, thread0_Main, 0, &D_801295E0, 255);
+    // It's possible D_801295E0 is a struct containing both the thread and stack pointer
```

Comment:
My guess is that `D_801295E0` is just the stack pointer, and the created `OSThread` is never used after started, so it's just being allocated on the given stack instead.